.github/workflows/CI.yaml

```
@@ -0,0 +1,40 @@
+      - uses: actions/checkout@v4.1.1
+
+      - name: Set up Python 3.13
+        uses: actions/setup-python@v5.0.0
+        with:
+          python-version: "3.13"
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install pytest
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)